### PR TITLE
install.sh fixes

### PR DESCRIPTION
**Pull request by harshdoesdev**
Fixes the below mentioned issues in the `install.sh` file:
- Removes any previously downloaded temporary binary files

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -77,6 +77,16 @@ update_path() {
     fi
 }
 
+remove_existing_files() {
+    local existing_files=(fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64)
+
+    for file in "${existing_files[@]}"; do
+        if [ -f "$file" ]; then
+            rm -f "$file"
```
Comment (amitu) [inline install.sh:85]: Isn't `rm fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64` enough?

Comment (amitu) [reply install.sh:85]: You are trying to ignore the error message?

Comment (harshdoesdev) [reply install.sh:85]: yes, maybe the installation failed because insufficient permissions, and the user runs the installer again

Comment (harshdoesdev) [reply install.sh:85]: thanks, changed it to: `rm -f fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64`

**Review by Heulitig — APPROVED** [review]

**Pull request merged**